x/rollapp/client/cli/tx_update_rollapp.go

```diff
@@ -12,7 +12,7 @@ import (
 
 func CmdUpdateRollapp() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "update-rollapp [rollapp-id] [init-sequencer] [genesis_checksum] [metadata]",
+		Use:     "update-rollapp [rollapp-id] [init-sequencer] [genesis_checksum] [metadata] [bech32-prefix]",
```

Comment:
I'd position bech32 before the metadata as it's required

Reply:
done